src/assets/js/sfu/room.js

```diff
@@ -19,6 +19,7 @@ export default class Room extends EventEmitter {
             var SFU_URL = "wss://" + window.location.hostname + ":2345";
             console.log("Joining Local SFU", SFU_URL);
             const wsTransport = new WebSocket(`${SFU_URL}/?roomId=${room}&peerId=${peerId}`, "protoo");
+            if(!wsTransport.onopen){throw 'local connection not working'};
```

Comment:
It should throw, and come in the failback situation of weird this did not happen with you